bingo.js

```diff
@@ -240,39 +258,53 @@ function loadSettings()
 {
 	getSettingsFromURL();
 	getSettingsFromLocalStorage();
+	if (! loadProgressFromURL()) {
+		loadProgressFromLocalStorage();
+	}
```

Comment:
These changes in `loadSettings()` would conflict with 6d28235a57a3c7cbe481b02d08552b7a3209318e. I'll drop the commit from #117 and deal with the issue brought up in the commit message later.